lib/tbot/spiffe/trust_bundle_cache.go

```diff
@@ -576,6 +586,11 @@ func listAllSPIFFEFederations(
 			PageToken: token,
 		})
 		if err != nil {
+			// Support auth server being too old.
```

Comment:
Before `FetchInitialBundleSet` was introduced, as part of the cache init, it would determine if the auth server was new enough, and then conditionally call `listAllSPIFFEFederations`. However, since introducing `FetchInitialBundleSet`, there's now no clean place to perform that check ahead of time, and it's safe to fail to just not returning anything here.